crates/holochain/tests/inline_zome_spec.rs

```diff
@@ -101,10 +132,10 @@ async fn inline_zome_3_agents_2_dnas() -> anyhow::Result<()> {
 
     let hash_foo: HeaderHash = conductor
         .call(&alice_foo.zome("foozome"), "create_unit", ())
-        .await;
+        .await?;
```

Comment:
The only issue with using results like this in tests is it can be hard to tell which call failed